test/integration.js

```diff
@@ -55,10 +52,15 @@ describe('integrations', function() {
     }
 
     pipe([
-      vfs.src(inputGlob, { buffer: false }),
+      vfs.src(inputGlob, { highWaterMark: expectedCount }),
```

Comment:
I could not find `buffer` in use. `highWaterMark` is not a documented option, but is used in code and other tests, so I went with it.